breathecode/registry/views.py

```diff
@@ -414,6 +422,10 @@ def get(self, request, asset_slug=None):
             param = self.request.GET.get('technologies')
             lookup['technologies__in'] = [p.lower() for p in param.split(',')]
 
+        if 'keywords' in self.request.GET:
+            param = self.request.GET.get('keywords')
+            items = items.filter(seo_keywords__slug__in=[p.lower() for p in param.split(',')])
```

Comment:
Remove the `.lower()` because the django field does not validate the uppercase and is possible has a slug with uppercase